setup.py

```diff
@@ -1,8 +1,11 @@
-# Copyright 2015 Knowledge Economy Developments Ltd
+# Copyright 2017 Henry Gomersall and Frederik Beaujean
```

Comment:
Ok with me. I don't insist on having my name in there. Just wanted to make sure people can find me if they have questions about what and why. Perhaps there should be a list of contributors at the end of the docs that list the PyFFTW contributors?